nipype/pipeline/plugins/tools.py

```diff
@@ -26,17 +26,29 @@ def report_crash(node, traceback=None, hostname=None):
     """
     name = node._id
     host = None
-    if node.result and getattr(node.result, 'runtime'):
```

Comment:
we seem to have lost the ability to check if `runtime` is part of result. i think at this point, `node.result` can be `None`

Reply:
Should be fine now